# Crie uma função chamada mostrar_dobro que receba um número.

def mostrar_dobro(x):
    resultado = x * 2
    print(resultado)
mostrar_dobro(7)

# Caso a variavel estiver fora da função, ocorrera o erro: Exception has occurred: NameError